Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -886,4 +886,19 @@ final class AttributeTests: ParserTestCase {
       """
     )
   }
+
+  func testIsolatedTypeAttribute() {
+    assertParse(
+      """
+      var fn: @isolated(any) () -> ()
+      """
+    )
+
+    // We don't validate the kind in the parser
+    assertParse(
+      """
+      var fn: @isolated(sdfhsdfi) () -> ()
+      """
+    )
```

Comment:
Can you combine `@isolated` with other attributes? Might be worth adding a test case that has two attributes eg. `@isolated` and `@convention`? Thinking of this because you added the `@isoalted` check after `@convention` in `skipTypeAttribute`.

Reply:
Added a test specifically testing the combination with `@convention`.